lib/Ohmbrewer_Onewire.cpp

```diff
@@ -99,17 +101,111 @@ double Ohmbrewer::Onewire::getReading(){
 }
 
 /**
+ * outputs probe IDs and their current temperatures to the screen
+ *
+ * Primarily for use as a manual identification tool for probe IDs
+ *
+ * FIXME probeID output is currently in HEX something or other, *non readable*
+ */
+void Ohmbrewer::Onewire::displayProbeIds(Ohmbrewer::Screen *screen){
+    //header
+    screen->resetTextSize();
+    screen->resetTextColor();
+    // Print the section title
+    screen->print("PROBE ID# : TEMP");
+//    screen->println("");
+    screen->printMargin(2);
+
+
+    uint8_t sensors[80];
+    uint8_t subzero, cel, celFracBits;        //local vars
+    double tempC;
+    //Asks all DS18x20 devices to start temperature measurement, takes up to 750ms at max resolution
+    DS18X20_start_meas( DS18X20_POWER_PARASITE, NULL );
+    //If your code has other tasks, you can store the timestamp instead and return when a second has passed.
+    delay(1000);
+    uint8_t numSensors = ow_search_sensors(10, sensors);
+    screen->displayHeader();
+    for (uint8_t i=0; i<numSensors; i++){
+        //current probe ID
+        char probeId[8] = {
+                sensors[(i*OW_ROMCODE_SIZE)+0],
+                sensors[(i*OW_ROMCODE_SIZE)+1],
+                sensors[(i*OW_ROMCODE_SIZE)+2],
+                sensors[(i*OW_ROMCODE_SIZE)+3],
+                sensors[(i*OW_ROMCODE_SIZE)+4],
+                sensors[(i*OW_ROMCODE_SIZE)+5],
+                sensors[(i*OW_ROMCODE_SIZE)+6],
+                sensors[(i*OW_ROMCODE_SIZE)+7]};
+
+        if (sensors[i*OW_ROMCODE_SIZE+0] == 0x10 || sensors[i*OW_ROMCODE_SIZE+0] == 0x28){ //0x10=DS18S20, 0x28=DS18B20
+           //for each probe print probe and temp to screen
+            if ( DS18X20_read_meas( &sensors[0], &subzero, &cel, &celFracBits) == DS18X20_OK ) {
+                //char sign = (subzero) ? '-' : '+';
+                int frac = celFracBits*DS18X20_FRACCONV;
+                tempC = (double)cel;
+                tempC = tempC + (.0001*(double)frac);
+
+                //print the ID and temp TODO fix me! hex value?
+                screen->print(probeId);
+//                screen->print(probeId[0]);
+//                screen->print(probeId[1]);
+//                screen->print(probeId[2]);
+//                screen->print(probeId[3]);
+//                screen->print(probeId[4]);
+//                screen->print(probeId[5]);
+//                screen->print(probeId[6]);
+//                screen->print(probeId[7]);
```

Comment:
Alrighty, then leave it.

I had a comment in here suggesting breaking the calls to screen->print into a separate method so you don't have to repeat it, but it seems to have disappeared. You should still do that - something like:
```c++
methodName(id, temp) {
screen->print(id);
screen->print(":");
screen->println(temp);
}
```
(approximately, of course)